From b3ede54f2699ec78fc88b9f59330fc905ac38b8e Mon Sep 17 00:00:00 2001
From: Sylwester Nawrocki <s.nawrocki@samsung.com>
Date: Fri, 13 Sep 2013 12:50:21 +0200
Subject: [PATCH 0785/1302] exynos4-is: Make fimc-lite dependency on fimc-is
 explicit in Kconfig

For power/clock management reasons FIMC-LITE devices can be used only
when the FIMC-IS driver is initialized.

Signed-off-by: Sylwester Nawrocki <s.nawrocki@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/media/platform/exynos4-is/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/exynos4-is/Kconfig b/drivers/media/platform/exynos4-is/Kconfig
index d2c670c..7302cd7 100644
--- a/drivers/media/platform/exynos4-is/Kconfig
+++ b/drivers/media/platform/exynos4-is/Kconfig
@@ -40,8 +40,7 @@ if SOC_EXYNOS4212 || SOC_EXYNOS4412 || SOC_EXYNOS5250
 
 config VIDEO_EXYNOS_FIMC_LITE
 	tristate "EXYNOS FIMC-LITE camera interface driver"
-	depends on I2C
-	select VIDEOBUF2_DMA_CONTIG
+	depends on VIDEO_EXYNOS4_FIMC_IS
 	select VIDEO_EXYNOS4_IS_COMMON
 	help
 	  This is a V4L2 driver for Samsung EXYNOS4/5 SoC FIMC-LITE camera
@@ -54,6 +53,7 @@ endif
 config VIDEO_EXYNOS4_FIMC_IS
 	tristate "EXYNOS4x12 FIMC-IS (Imaging Subsystem) driver"
 	select VIDEOBUF2_DMA_CONTIG
+	select VIDEO_EXYNOS_FIMC_LITE
 	depends on OF
 	select FW_LOADER
 	help
-- 
1.8.3.2

